packages/squiggle-lang/__tests__/Distributions/Invariants/Means_test.res

```diff
@@ -136,20 +136,20 @@ describe("Means are invariant", () => {
     )
 
     testAll("with two of the same distribution", distributions, dist => {
-      E.R.liftM2(testMultiplicationMean, dist, dist)->E.R.toExn
+      E.R.liftM2(testMultiplicationMean, dist, dist)->E.R.toExn("Means were not invariant", _)
     })
 
     testAll("with two different distributions", pairsOfDifferentDistributions, dists => {
       let (dist1, dist2) = dists
-      E.R.liftM2(testMultiplicationMean, dist1, dist2)->E.R.toExn
+      E.R.liftM2(testMultiplicationMean, dist1, dist2)->E.R.toExn("Means were not invariant", _)
     })
 
     testAll(
       "with two different distributions in swapped order",
       pairsOfDifferentDistributions,
       dists => {
         let (dist1, dist2) = dists
-        E.R.liftM2(testMultiplicationMean, dist2, dist1)->E.R.toExn
+        E.R.liftM2(testMultiplicationMean, dist2, dist1)->E.R.toExn("Means were not invariant", _)
```

Comment:
This file wasn't particularly elegant before, but now is as good a time as any to just make a function for this one line. The text shouldn't be duplicated so many places, but really, this entire line could be turned into a function.